tests/network-ovn

```diff
@@ -22,6 +23,7 @@ if [ "$#" -gt 0 ] && [ "$1" = "microovn" ]; then
     snap restart microovn
     microovnAddr=$(microovn cluster list --format json | jq -r '.[0].address | split(":") | first')
     lxc config set network.ovn.northbound_connection "ssl:${microovnAddr}:6641"
+    backend="microovn."
```

Comment:
It might be simpler to create the alias instead:
```
snap alias microovn.ovn-nbctl ovn-nbctl
```

This makes the command available as just `ovn-nbctl`.